server/src/main/java/tw/waterball/survive/io/server/Client.java

```diff
@@ -1,17 +1,21 @@
 package tw.waterball.survive.io.server;
 
-import tw.waterball.survive.io.domain.model.gameobject.GameObject;
-import tw.waterball.survive.io.domain.model.gameobject.role.Player;
-import tw.waterball.survive.io.domain.model.gameobject.role.Role;
-import tw.waterball.survive.io.domain.model.gameobject.weapon.Weapon;
+import tw.waterball.survive.io.server.worldobject.WorldObject;
+import tw.waterball.survive.io.server.worldobject.role.Player;
+import tw.waterball.survive.io.server.worldobject.role.Role;
+import tw.waterball.survive.io.server.worldobject.weapon.Weapon;
 
 public interface Client {
 
-    void onLoginSuccess();
+    void onLoginSuccessfully();
 
     void onHeartBeat();
 
-    void onSufferJury(Role role, int damage);
+    void onMoveVertical(Role role);
```

Comment:
no, how should you come to this? Tell nothing but move vertically? How does this work? How does client understand the direction of he moving?